test/info/info.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { sep } = require('path');
+const { run, extractSummary } = require('../utils/test-utils');
+const pkgJSON = require('../../package.json');
+
+describe('info flag', () => {
+    it('outputs version of webpack', () => {
```

Comment:
This test isn't aimed at `info` command. Version is provided by `--version` flag.